### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by JonathanLorimer**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,7 +14,12 @@ run mdformat on your own. CI will fail if markdown files are not correctly
 formatted!
 
 - [mdbook](https://rust-lang.github.io/mdBook/cli/index.html)
-- [mdformat](https://pypi.org/project/mdformat/)
+- [prettier](https://prettier.io/)
```
Comment (srirajshukla) [inline README.md:17]: It still says "run mdformat" in line 13.

File: README.md [hunk 2]
```diff
@@ -13,7 +14,12 @@ run mdformat on your own. CI will fail if markdown files are not correctly
 formatted!
```
Comment (srirajshukla) [inline README.md]: It says 'run mdformat' here, shouldn't it be run Prettier?

Comment (JonathanLorimer) [reply README.md]: Good catch, updated

Comment (srirajshukla) [reply README.md]: Can we add a link for how to enable experimental features (if someone wants to)?

Comment (JonathanLorimer) [reply README.md]: Sure, I think that is kind of marginal (the docs here shouldn't be responsible for teaching someone how to use nix, they just offer a method of setting up the project for nix users) but I am happy to provide a link.

**Review by Mikolaj — APPROVED** [review]
I'm not an expert in this area, but I see nothing offending. :)

**Review by emilypi — APPROVED** [review]

**Pull request merged**